test/test.json

```diff
@@ -1,35 +1,35 @@
-{
-    "messages": [
-        {
-            "id": 291,
-            "isextended_frame": false,
-            "name": "testFrame1",
-            "signals": [
-                {
-                    "bit_length": 11,
-                    "factor": 5.0,
-                    "is_big_endian": true,
-                    "is_signed": false,
-                    "name": "someTestSignal",
-                    "offset": 1.0,
-                    "start_bit": 9
-                },
-                {
-                    "bit_length": 3,
-                    "factor": 1.0,
-                    "is_big_endian": false,
-                    "is_signed": false,
-                    "name": "Signal",
-                    "offset": 0.0,
-                    "start_bit": 20
-                }
-            ]
-        },
-        {
-            "id": 18,
-            "isextended_frame": true,
-            "name": "extendedFrame",
-            "signals": []
-        }
-    ]
+{
+    "messages": [
+        {
+            "id": 291,
+            "isextended_frame": false,
+            "name": "testFrame1",
+            "signals": [
+                {
+                    "bit_length": 11,
+                    "factor": 5.0,
+                    "is_big_endian": true,
+                    "is_signed": false,
+                    "name": "someTestSignal",
+                    "offset": 1.0,
+                    "start_bit": 9
+                },
+                {
+                    "bit_length": 3,
+                    "factor": 1.0,
+                    "is_big_endian": false,
+                    "is_signed": false,
+                    "name": "Signal",
+                    "offset": 0.0,
+                    "start_bit": 20
+                }
+            ]
+        },
+        {
+            "id": 18,
+            "isextended_frame": true,
+            "name": "extendedFrame",
+            "signals": []
+        }
+    ]

```

Comment:
Hmm, is this just newline changes?  It would be nice if github showed a bit better...